jaeger_client/TUDPTransport.py

```diff
@@ -26,30 +27,31 @@ class TUDPTransport(TTransportBase, object):
     TUDPTransport implements just enough of the tornado transport interface
     to work for blindly sending UDP packets.
     """
-    def __init__(self, host, port, blocking=False):
+    def __init__(self, host: str, port: int, blocking: bool = False) -> None:
         self.transport_host = host
         self.transport_port = port
-        self.transport_sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-        if blocking:
-            blocking = 1
-        else:
-            blocking = 0
+        self.transport_sock: Optional[socket.socket] = (
+            socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
```

Comment:
This can return None?

Reply:
Nope, but the variable can get None. If you call close, you set the variable to None